src/providers/pdal/CMakeLists.txt

```diff
@@ -111,6 +111,8 @@ set_target_properties(untwine PROPERTIES
   RUNTIME_OUTPUT_DIRECTORY ${QGIS_OUTPUT_DIRECTORY}/${QGIS_LIBEXEC_SUBDIR}
 )
 
+target_compile_definitions(untwine PRIVATE WIN32_LEAN_AND_MEAN)
```

Comment:
already in line 5.

IOW c5ea4b220cbbd7c7a56b7c4cea59bfce454a0c05 should already cover this fully.